lib/phoenix/transports/websocket.ex

```diff
@@ -22,7 +22,13 @@ defmodule Phoenix.Transports.WebSocket do
       origin header is present. It defaults to true and, in such cases,
       it will check against the host value in `YourApp.Endpoint.config(:url)[:host]`.
       It may be set to `false` (not recommended) or to a list of explicitly
-      allowed origins
+      allowed origins.
+
+      check_origin: ["https://example.com",
+                     "//another.com:888", "//other.com"]
+
+      Note: it is necessary to set `check_origin: false` to support native
```

Comment:
This is not true, a native app can set that it is connecting 'from' an origin anyway, even when it is not.  Maybe instead word it as:
```markdown
Note: To connect from a native app be sure to either have the native app set an origin or allow any origin via `check_origin: false`
```